### PR TITLE
Build header update

### PR DESCRIPTION
**Pull request by PixelJanitor**

### CODE REVIEW

**Comment by PixelJanitor** [comment]
- [ ] Pending states
- [ ] Comparison Icon component that uses build state

File: app/models/build.js [hunk 1]
```diff
@@ -68,6 +68,10 @@ export default DS.Model.extend({
   // Each comparison represents a single individual rendering at a width and browser.
   totalSnapshots: DS.attr('number'),
   totalSnapshotsUnreviewed: DS.attr('number'),
+  totalSnapshotsApprovedByUser: computed('totalSnapshots', 'totalSnapshotsUnreviewed', function() {
```
Comment (fotinakis) [inline app/models/build.js]: Unused, drop this?

File: app/templates/components/build-overview.hbs [hunk 2]
```diff
@@ -1,68 +1,40 @@
...
+    {{#if build.isPending}}
+      Receiving build
+    {{else if build.isProcessing}}
+      Processing visual differences
+    {{else if build.isFinished}}
       {{#if build.isApproved}}
-        were <strong>approved</strong>
-        {{#if build.approvedBy}}
-          <span class="ApprovalSection-user">by {{build.approvedBy.name}}</span>
+        {{#if build.hasNoDiffs}}
```
Comment (fotinakis) [inline app/templates/components/build-overview.hbs]: Can't use `hasDiffs` or `hasNoDiffs` logically here — "hasDiffs" will still be true for a build, even if all of them are approved. We treat "having diffs" and "having unreviewed snapshots" as different things.

If you ever check `build.isApproved` and then need to switch on something else, likely you should be checking `build.reviewStateReason` next and doing:

```hbs
{{#if (eq build.reviewStateReason 'all_snapshots_approved')}}
{{else if (eq build.reviewStateReason 'all_snapshots_approved_previously')}}
{{else if (eq build.reviewStateReason 'no_diffs')}}
{{/if}}
```

File: app/models/build.js (hunk 1 above)
Comment (PixelJanitor) [reply app/models/build.js]: Nice catch. This was here and previously used as a placeholder.

**Comment by timhaines** [comment]
Take a look inside the build info.  Is all in a big font.

**Comment by PixelJanitor** [comment]
Font sizing fixed!

**Comment by fotinakis** [comment]
@PixelJanitor added a commit here, just want to enumerate all the review_state_reasons because we will likely expand them over time. https://github.com/percy/percy-web/pull/421/commits/ec94ac197fd937616afc6e12403abb3206403970

**Pull request merged**